foreigndonors.php

```diff
@@ -219,24 +219,32 @@ function foreigndonors_civicrm_buildForm($formName, &$form) {
     }
 
     $isNSW = substr($finType[0]['financial_type_id:label'], 0, 3) === 'NSW';
+    $isACT = substr($finType[0]['financial_type_id:label'], 0, 3) === 'ACT';
 
     // Add the checkbox to the public form
     // Have to use different language for Queensland
-    // Inject a prohibited donor checkbox for NSW
-    // if the domain is NSW or the fin type starts with 'NSW'
-    if ($domainId == 7 && !$isNSW) {
+    // Inject a prohibited donor checkbox for NSW and ACT
+    if ($domainId == 7 && !$isNSW && !$isACT) {
       $label = "I am an Australian Citizen or Permanent Resident, and not a QLD prohibited donor";
       $form->add('Checkbox', 'foreigndonor', ts('I am an Australian Citizen or Permanent Resident, and not a QLD prohibited donor'));
       $form->addRule('foreigndonor', ts('You must affirm you are not a prohibited donor as per State and Federal legislation'), 'required', NULL, 'client');
     }
     elseif ($domainId == 8 || $isNSW ) {
-      $form->assign('addProhibitedDonor', TRUE);
+      $form->assign('addNSWProhibitedDonor', TRUE);
       $label = "I am an Australian Citizen or Permanent Resident and am not a foreign donor";
       $form->add('Checkbox', 'foreigndonor', ts('I am an Australian Citizen or Permanent Resident and am not a foreign donor'));
       $form->addRule('foreigndonor', ts('You must affirm you are an Australian Citizen or Permanent Resident'), 'required', NULL, 'client');
       $form->add('Checkbox', 'prohibiteddonor', ts('I am not a prohibited donor as per NSW Electoral Legislation'));
       $form->addRule('prohibiteddonor', ts('You must affirm you are not a prohibited donor as per NSW Electoral legislation'), 'required', NULL, 'client');
     }
+    elseif ($domainId == 9 || $isACT ) {
+      $form->assign('addACTProhibitedDonor', TRUE);
+      $label = "I am an Australian Citizen or Permanent Resident and am not a foreign donor";
+      $form->add('Checkbox', 'foreigndonor', ts('I am an Australian Citizen or Permanent Resident and am not a foreign donor'));
+      $form->addRule('foreigndonor', ts('You must affirm you are an Australian Citizen or Permanent Resident'), 'required', NULL, 'client');
+      $form->add('Checkbox', 'prohibiteddonor', ts('I am not a prohibited donor as per ACT Electoral Legislation'));
+      $form->addRule('prohibiteddonor', ts('You must affirm you are not a prohibited donor as per ACT Electoral legislation'), 'required', NULL, 'client');
```

Comment:
(minor) Upper case "l" in "legislation" to match preceeding line.

Reply:
Or maybe I should just have "ACT electoral legislation", i.e. avoid unnecessary capitals.  I just copied and pasted what NSW had, so I could do it there too.